proposals/text/README.md

```diff
@@ -96,6 +96,7 @@ The primvars:displaycolor and the primvars:displayopacity will be the color and
 - textData. The text data is a UTF-8 string. This property must be specified for SimpleText.
 - primvars:backgroundColor. The color of the background. By default there is no background color. (Currently this not supported in our implementation. Will be implemented in the future.)
 - primvars:backgroundOpacity. The opacity of the background. By default there is no background opacity. (Currently this not supported in our implementation. Will be implemented in the future.)
+- textMetricsUnit. The unit for textHeight and other text metrics. It is a string token, which could be "pixel", "publishingPoint" or "worldUnit". By default it is "worldUnit". If the text primitive doesn't have billboard, only "worldUnit" is valid. If the unit is set to other value, we will still use "worldUnit" unit. The unit of the textHeight will be the same as the unit of the world space. If the text primitive has billboard, that it always faces to the camera and the size will not change, only the value "pixel" and "publishingPoint" are effective. If metrics unit is "pixel", the unit will be pixel on the screen. If metrics unit is "publishingPoint", the unit will be desktop publishing point or 1/72 inch on the screen. If the unit is set to "worldUnit", we will use "pixel" as the value. 
```

Comment:
Is this a correct reading?

- textMetricsUnit. The unit for textHeight and other text metrics. It is a string token, which could be "pixel", "publishingPoint" or "worldUnit". By default it is "worldUnit".

- publishingPoint and pixel are only valid if the text primitive's billboard property is true

- text height unit is always world space unit, irrespective of this setting. Therefore textMetric is for positioning only. In the case of billboards, this may result in overlapping lines of text depending on camera position.

- billboarded text primitives always face the camera and are a fixed size, so textMetricsUnit must not be set to worldUnit for billboarded text.

- pixels are pixels in the framebuffer.

- publishingPoint are 1/72 of an inch on a screen's physical display. Achieving this requires a renderer to know the physical dimensions of a monitor.

- if textMetricsUnit is set to "worldUnit", "pixel" is used regardless, so the "worldUnit" setting has no application.

Is this correct? If not, can we rewrite the section to make it more clear?

Reply:
> Is this a correct reading?
> 
> * textMetricsUnit. The unit for textHeight and other text metrics. It is a string token, which could be "pixel", "publishingPoint" or "worldUnit". By default it is "worldUnit".
> * publishingPoint and pixel are only valid if the text primitive's billboard property is true
> * text height unit is always world space unit, irrespective of this setting. Therefore textMetric is for positioning only. In the case of billboards, this may result in overlapping lines of text depending on camera position.
> * billboarded text primitives always face the camera and are a fixed size, so textMetricsUnit must not be set to worldUnit for billboarded text.
> * pixels are pixels in the framebuffer.
> * publishingPoint are 1/72 of an inch on a screen's physical display. Achieving this requires a renderer to know the physical dimensions of a monitor.
> * if textMetricsUnit is set to "worldUnit", "pixel" is used regardless, so the "worldUnit" setting has no application.
> 
> Is this correct? If not, can we rewrite the section to make it more clear?

"If the text primitive doesn't have billboard, only "worldUnit" is valid. If the unit is set to other value, we will still use "worldUnit" unit. The unit of the textHeight will be the same as the unit of the world space." This should be a separate paragraph. The unit of text Height is the unit of world space, when the text primitive doesn't have billboard. 

Yes, I think I need to rewrite the section so that it could be more clear..